Lapka.Pets.Application/Exceptions/ValueNotFoundException.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Lapka.Pets.Application.Exceptions
+{
+    public class PetNotFoundException : AppException
+    {
+        public override string Code => "pet_not_found";
```

Comment:
Add here Guid for which you are querying. Add it to the message and create property with this id